content/en-us/reference/engine/globals/LuaGlobals.yaml

```diff
@@ -214,6 +214,12 @@ functions:
 
       `loadstring()` is disabled by default. For guidance around enabling it,
       see `Class.ServerScriptService`.
+
+      **WARNING:**  This method disables certain Luau optimizations on the returned function.
```

Comment:
Currently, `<Alert>` annotations do not render as expected in API docs, so I put this into normal text.